docs/src/main/sphinx/installation/plugins.md

```diff
@@ -0,0 +1,311 @@
+# Plugins 
+
+Trino uses a plugin architecture to extend its capabilities and integrate with
+various data sources and other systems. Trino includes many plugins as part of
+the binary packages - specifically the `tar.gz` archive and the Docker
+container.
+
+Plugins implement some of the following capabilities:
+
+* [Connectors](/connector),
+* [Authentication types](security-authentication)
+* [Access control systems](security-access-control)
+* [Event listeners](admin-event-listeners)
+* Additional types, functions, and table functions
```

Comment:
Table functions are actually part of connectors. How about

> Additional types and global functions

Reply:
done